backend/de.metas.async/src/main/java/de/metas/async/api/impl/AsyncBatchBL.java

```diff
@@ -165,10 +165,15 @@ public void increaseProcessed(final I_C_Queue_WorkPackage workPackage)
 
 	@Override
 	public void enqueueAsyncBatch(@NonNull final AsyncBatchId asyncBatchId)
+	{
+		enqueueAsyncBatch0(asyncBatchId);
+	}
+
+	private void enqueueAsyncBatch0(
```

Comment:
what's the reason of introducing enqueueAsyncBatch0?

Reply:
removed